src/clockcache.c

```diff
@@ -918,7 +943,7 @@ clockcache_assert_no_refs(clockcache *cc)
 {
    threadid        i;
    volatile uint32 j;
-   for (i = 0; i < (MAX_THREADS - 1); i++) {
+   for (i = 0; i < MAX_THREADS; i++) {
```

Comment:
1st instance of an off-by-1 check.

Reply:
The history of these is that we used to re-use the `(MAX_THREADS - 1)` counter for pins.